package/kernel/linux/modules/other.mk

```diff
@@ -135,6 +135,18 @@ endef
 $(eval $(call KernelPackage,btmrvl))
 
 
+define KernelPackage/cpuid
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=Intel CPUID support
+  KCONFIG:=CONFIG_X86_CPUID
+  DEPENDS:=@TARGET_x86
+  FILES:=$(LINUX_DIR)/arch/x86/kernel/cpuid.ko
+  AUTOLOAD:=$(call AutoLoad,20,cpuid)
+endef
+
```

Comment:
This is activated by default for target/linux/x86/geode/config-default
Should this also be activated by default for more x86 configurations?

Reply:
> Should this also be activated by default for more x86 configurations?

@Hauke it's not a big driver and more processors support it than not.